test/test_device_histogram.cu

```diff
@@ -44,8 +44,14 @@
 
 #include "test_util.h"
 
-using namespace cub;
+#define TEST_HALF_T \
+  (__CUDACC_VER_MAJOR__ >= 9 || CUDA_VERSION >= 9000) && !_NVHPC_CUDA
```

Comment:
Question: Why `!_NVHPC_CUDA`?

Reply:
> ────────────────────────────────────────────────┐
    commit e882cfd875417589276586f4657f285885c523b9 │
   ────────────────────────────────────────────────┘
    Author: David Olsen <dolsen@nvidia.com>
    Date:   Thu Nov 5 14:39:01 2020 -0800
    Fix cub/util_type.cuh to not use 16-bit float with NVC++
    A recent commit changed several occurances of "__CUDACC_VER_MAJOR__ >= 9"
    in preprocessor expressions to "(__CUDACC_VER_MAJOR__ >= 9 ||
    CUDA_VERSION >= 9000)".  NVC++ defines CUDA_VERSION but not
    __CUDACC_VER_MAJOR__, so the commit changed the condition from false to
    true for NVC++.  The code inside those blocks doesn't work with NVC++,
    causing compilation errors.  Fix the regression by changing the expressions
    to "(__CUDACC_VER_MAJOR__ >= 9 || CUDA_VERSION >= 9000) &&
    !__NVCOMPILER_CUDA__".